### PR TITLE
Fix logo issue - Transmit Security

### PR DESCRIPTION
**Pull request by tomzarhin5**
Required items, please complete
   
   Change(s):
   - See guidance below

   Reason for Change(s):
   - See guidance below

   Version Updated:
   - Required only for Detections/Analytic Rule templates
   - See guidance below

   Testing Completed:
   - See guidance below

   Checked that the validations are passing and have addressed any issues that are present:
   - See guidance below

# Guidance <- remove section before submitting
-----------------------------------------------------------------------------------------------------------
## Before submitting this PR please ensure that you have read the following sections and filled out the changes, reason for change and testing complete sections:

Thank you for your contribution to the Microsoft Sentinel Github repo.

> Details of the code changes in your submitted PR.  Providing descriptions for pull requests ensures there is context to changes being made and greatly enhances the code review process.  Providing associated Issues that this resolves also easily connects the reason.
   
   Change(s):
   - Updated syntax for XYZ.yaml

   Reason for Change(s):
   - New schema used for XYZ.yaml
   - Resolves ISSUE #1234

   Version updated:
   - Yes
   - Detections/Analytic Rule templates are required to have the version updated

> The code should have been tested in a Microsoft Sentinel environment that does not have any custom parsers, functions or tables, so that you validate no incorrect syntax and execution functions properly.  If your submission requires a custom parser or function, it must be submitted with the PR.

   Testing Completed:
   - Yes/No/Need Help

_Note: If updating a detection, you must update the version field._

> Before the submission has been made, please look at running the KQL and Yaml Validation Checks locally.
> https://github.com/Azure/Azure-Sentinel#run-kql-validation-locally

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes/No/Need Help
   
   _Note: Let us know if you have tried fixing the validation error and need help._

> References: 
> - [Guidance for Detection checks](https://github.com/Azure/Azure-Sentinel#pull-request-detection-template-structure-validation-check)
> - [General contribution guidance](https://github.com/Azure/Azure-Sentinel/wiki#what-can-you-contribute-and-how-can-you-create-contributions)
> - [PR validation troubleshooting](https://github.com/Azure/Azure-Sentinel#pull-request)

-----------------------------------------------------------------------------------------------------------

### CODE REVIEW

**Comment by tomzarhin5** [comment]
Hi, I received feedback while uploading the package zip for our Transmit Security offer on Microsoft Partners. Here are the comments:
	•	The Search Keywords for Sentinel GUID (f1de974b-f438-4719-b423-8bf704ba2aef) should be added only once. It appears to be added three times. Instead, it is suggested to use “Sentinel” and the offer name for the other two instances of the search keywords.
	•	The logo is breaking. Please correct this issue.

I have fixed the logo issue, but I’m unclear about the first comment regarding the search keywords. What specifically needs to be done in this Pull Request to resolve it?

**Comment by tomzarhin5** [comment]
Hi @v-prasadboke ,
Could you please see my question above?
Thank you

**Comment by v-prasadboke** [comment]
Hello @tomzarhin5, Sorry for the delay in response 

Got this points earlier 

Will be reviewing this PR

**Comment by v-prasadboke** [comment]
Hello @tomzarhin5, Can you provide write access to your branch 

need to commit some required changes

**Comment by tomzarhin5** [comment]
Hi @v-prasadboke,
I apologize, but I’m unable to proceed as I don’t have admin access to my forked repository within the organization. Could we schedule a short meeting where you could guide me on the required fix?

File: Solutions/TransmitSecurity/Data/Solution_TransmitSecurity.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
 {
     "Name": "TransmitSecurity",
     "Author": "Transmit Security",
-    "Logo": "<img src=\"https://portal.transmitsecurity.io/assets/logo-c16ca289.svg\" width=\"75px\" height=\"75px\">",
```
Comment (v-prasadboke) [inline Solutions/TransmitSecurity/Data/Solution_TransmitSecurity.json:5]: replace logo link with this link 

"",

Comment (tomzarhin5) [reply Solutions/TransmitSecurity/Data/Solution_TransmitSecurity.json:5]: Done

**Review by v-prasadboke — APPROVED** [review]

**Pull request merged**